uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/controllers/modal/modal-document.js

```diff
@@ -261,7 +262,7 @@ define([
           } else if (result.files.length && !!result.files[0].values[0].result) {
             DocumentService.save({
               id: result.document.id,
-              status_id: '3' // 3 => 'approved'
+              status_id: ($scope.isRole('admin')) ? '3' : '1' // 3 => 'approved', 1 => 'awaiting upload'
```

Comment:
1. `($scope.isRole('admin'))` the outer parenthesis are not necessary `$scope.isRole('admin')`
2. Why is the status id hardcoded?